src/js/components/award/visualizations/AmountsChart.jsx

```diff
@@ -170,89 +170,26 @@ export default class AmountsChart extends React.Component {
                     }}>
 
                     <g transform="translate(0, 5)">
```

Comment:
It depends on the situation. If you can, it would be best, but if the bars are heavily dependent on other components or their drawing calculation output needs to be used by other components, it may be better to keep it in the parent component.